scripts/geopmpy/launcher.py

```diff
@@ -1078,8 +1078,28 @@ def reservation_option(self):
             result = ['--reservation', self.reservation]
         return result
 
+    _static_performance_governor_option = None
     def performance_governor_option(self):
-        return ['--cpu-freq=Performance']
+        if SrunLauncher._static_performance_governor_option is None:
+            governor_file = '/sys/devices/system/cpu/cpu0/cpufreq/scaling_governor'
+            pid = subprocess.Popen(['srun', 'cat', '{}'.format(governor_file)], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
For init_topo() we had been careful to always carry over the user provided option:

```
                                  host_file=self.host_file,
                                  node_list=self.node_list,
                                  exclude_list=self.exclude_list,
                                  reservation=self.reservation)
```
Hopefully, the specific nodes we land on are not as important for the governor setting.